packages/experience/src/apis/api.ts

```diff
@@ -1,11 +1,17 @@
 import i18next from 'i18next';
 import ky from 'ky';
 
+import { searchKeys } from '@/utils/search-parameters';
+
 export default ky.extend({
   hooks: {
     beforeRequest: [
       (request) => {
         request.headers.set('Accept-Language', i18next.language);
+        const appId = sessionStorage.getItem(searchKeys.appId);
+        if (appId) {
+          request.headers.set('X-Logto-App-ID', appId);
```

Comment:
Should we put it in a shared const file?

Reply:
Updated README file in Logto maintained `oidc-provider` package, as per offline discussion.